src/main/java/com/bitso/websockets/BitsoStreamUpdate.java

```diff
@@ -0,0 +1,39 @@
+package com.bitso.websockets;
+
+import org.json.JSONObject;
+
+import com.bitso.BitsoBook;
+import com.bitso.exceptions.BitsoExceptionNotExpectedValue;
+import com.bitso.helpers.Helpers;
+
+public class BitsoStreamUpdate {
+    protected BitsoStreams bitsoStream;
+    protected BitsoBook bitsoBook;
+
+    public BitsoStreams getBitsoStream() {
+        return bitsoStream;
+    }
+        
+    public BitsoBook getBitsoBook() {
+        return bitsoBook;
+    }
+    
+    public BitsoStreamUpdate(JSONObject jsonObject){
+        bitsoStream = getStream(Helpers.getString(jsonObject, "type"));
+        bitsoBook = Helpers.getBook(Helpers.getString(jsonObject, "book"));
+    }
+    
+    protected BitsoStreams getStream(String stream) {
+        switch (stream) {
```

Comment:
BTW, instead of using a switch here, you can use something like:

protected <E extends Enum<E>> E getEnum(String key, Class<E> clazz) {
     return Enum.valueOf(clazz, val);
}